examples/never-type/never_type.description

```diff
@@ -0,0 +1,4 @@
+// The `never` type represents the type of values that never occur.
+// This can be useful to describe the return type of a function, if the function never returns.
+// You can use `never` type as a type parameter. As no value can ever belong to `never` type, you
```

Comment:
Didn't get what's meant by the following: "You can use `never` type as a type parameter"

Reply:
Did you mean the `xml` and `map` scenarios? Since we have the examples, IMO we don't need to say it here, since it's not a common term.

Reply:
Shall I remove that part and add it to the program file ?

Reply:
Or how about 

> Also, you can use `never` type as a type parameter., such as in `table` and `xml`

?

Reply:
Solved